pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -14,10 +14,12 @@
 , ncurses
 , libxml2
 , lit
+, llvm
 , filelock
 , torchWithRocm
 , python
 , cudaPackages
+, cudaSupport ? false
```

Comment:
`config.cudaSupport`?

Reply:
The top-level declaration already covers it.

```nix
openai-triton = callPackage ../development/python-modules/openai-triton {
  inherit (pkgs.config) cudaSupport;
  cudaPackages = pkgs.cudaPackages_12_0;
};
```

Reply:
actually I think I see what you mean now, it should be moved to the expression.